benchmark/benchmark.py

```diff
@@ -1,11 +1,14 @@
 import json
-import os
+from pathlib import Path
 import re
 import subprocess
 import sys
 
-sys.path.insert(0, os.path.abspath(
-     os.path.join(os.path.dirname(__file__), '..', 'test')))
+sys.path.insert(
+    0, str((Path(__file__).resolve().parent.parent / 'test')))
```

Comment:
Minor comment `.parent.parent` should be equivalent to`.parents[1]` I think. Not sure which one is better.